concepts/type-conversion/about.md

```diff
@@ -1,3 +1,43 @@
-# About
+# About Type Conversion
 
-TODO: add information on type-conversion concept
+In Go, assignment of a value between different types requires explicit conversion.
+For example, this is a compiler error:
+
```

Comment:
General comment: you might want to mention "type casting" as a term somewhere, e.g. like "this is also called ..." so the student knows what that means even if there previous language did not use the term.